package.json

```diff
@@ -2,7 +2,7 @@
 	"name": "gifsicle",
 	"version": "6.0.3",
 	"description": "gifsicle wrapper that makes it seamlessly available as a local dependency",
-	"license": "MIT",
+	"license": "GPL-2.0+",
```

Comment:
This should be GPL-2.0. The text says: GNU General Public License, Version 2 (and only Version 2). GPL-2.0+ means 2.0 or any later version.